extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/connection/ConnectionModel.java

```diff
@@ -55,6 +55,13 @@
     private Integer maxConnectionsPerUser;
 
     /**
+     * The weight of the connection for the purposes of calculating
+     * WRR algorithm.  null indicates nothing has been set, -1 indicates
```

Comment:
Yep...replaced a few instances of that, already, but missed this one.  Should be fixed.